apps/api/src/equipment/equipment.service.ts

```diff
@@ -0,0 +1,139 @@
+import { PrismaService } from "./../prisma/prisma.service"
+import { Injectable } from "@nestjs/common"
+import { CreateEquipmentDto } from "./dto/create-equipment.dto"
+import { UpdateEquipmentDto } from "./dto/update-equipment.dto"
+import { Prisma, EquipCategory } from "@repo/database"
+import { ConflictError, NotFoundError } from "@repo/api-client"
+import * as path from "node:path"
+import { randomUUID } from "node:crypto"
+import { equipmentWithRentalLogInclude } from "@repo/shared-types"
+
+@Injectable()
+export class EquipmentService {
+  constructor(private readonly prisma: PrismaService) {}
+
+  async create(
+    createEquipmentDto: CreateEquipmentDto,
+    file?: Express.Multer.File
+  ) {
+    const imageUrl: string | undefined = undefined
+    if (file) {
+      // 파일 업로드 필요.
+      const fileExt = path.extname(file.originalname)
+      const fileName = `${randomUUID()}${fileExt}`
+
+      // 파일 업로드 구현 필요
```

Comment:
```suggestion
      // TODO: 파일 업로드 구현 필요
```

<img width="144" height="348" alt="Image" src="https://github.com/user-attachments/assets/d1dcdda7-0727-47e1-bf4b-fb08a348e20b" />

<img width="93" height="71" alt="Image" src="https://github.com/user-attachments/assets/e215720d-0f15-4206-b2a2-f2a9b99dd9ac" />

할 일을 주석으로 남기는 것은 매우 좋은 습관이라고 생각합니다!(Good)
게다가 추후 [VSCode의 TODO Tree 확장](https://marketplace.visualstudio.com/items?itemName=Gruntfuggly.todo-tree) 사용 시 찾기 쉽게 `TODO` 키워드까지 추가해주시면 너무 좋을 것 같습니다!